### PR TITLE
[DOCFIX] Add Fuse Docker/Kubernetes docs

### PR DESCRIPTION
**Pull request by LuQQiu**
Alluxio more details docs for installing POSIX API via Docker/Kubernetes

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md [hunk 1]
```diff
@@ -1016,6 +1016,41 @@ $ helm upgrade alluxio -f config.yaml \
...
+  env:
+    MAX_IDLE_THREADS: "64"
+```
+
+[POSIX API docs]({{ '/en/api/POSIX-API.html' | relative_url }}) provides more details about how to configure Alluxio POSIX API.
+  {% endcollapsible %}
+{% endaccordion %}
+
 {% endnavtab %}
 {% navtab kubectl %}
```
Comment (LuQQiu) [inline docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1055]: @Xenorith Is it possible to embedd `navtab` and `accordion` layout together?

Comment (Xenorith) [reply docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1055]: yes you can put `accordion`s in `navtab`s. for example: https://docs.alluxio.io/ee/user/stable/en/hybrid-cloud/AWS-Advanced-Guide.html#specific-module-input-variables-and-outputs

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
LGTM. feel free to merge it when you feel ready

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**